item_library/templates/item_library/index.html

```diff
@@ -28,9 +38,9 @@
     </div>
     <div class="form-row">
       <div class="form-group">
-        <button type="submit" name="filter" class="btn btn-primary">Filter
+        <button type="submit" name="filter" id="btn" class="btn btn-primary">Filter
         </button>
-        <button type="submit" name="reset" class="btn btn-primary">Reset
+        <button type="submit" name="reset" id="btn" class="btn btn-primary">Reset
```

Comment:
Issue: Two different buttons have the same ``id``

Reply:
Thanks! My bad!